sycl-fusion/test/internalization/promote-local-vec.ll

```diff
@@ -30,59 +30,59 @@ declare spir_func void @__itt_offload_wi_start_wrapper() #3
+define spir_kernel void @fused_0(%0 addrspace(1)* align 16 %KernelOne_accTmp, ptr byval(%1) align 8 %KernelOne_accTmp3, %0 addrspace(1)* align 16 %KernelOne_accIn1, ptr byval(%1) align 8 %KernelOne_accIn16, %0 addrspace(1)* align 16 %KernelOne_accIn2, %0 addrspace(1)* align 16 %KernelTwo_accOut, %0 addrspace(1)* align 16 %KernelTwo_accIn3) !kernel_arg_addr_space !12 !kernel_arg_access_qual !13 !kernel_arg_type !14 !kernel_arg_type_qual !15 !kernel_arg_base_type !14 !kernel_arg_name !16 !sycl.kernel.promote !17 !sycl.kernel.promote.localsize !18 {
 ; Scenario: Test the successful private internalization of the first pointer
 ; argument. This means, the first pointer argument has been replaced by a
 ; function-local alloca and all accesses have been updated to use this alloca
 ; instead.
 
 ; CHECK-LABEL: define {{[^@]+}}@fused_0
-; CHECK-SAME: ([[TYPE0:%.*]] addrspace(3)* align 16 [[KERNELONE_ACCTMP:%.*]], %1* byval([[TYPE1:%.*]]) align 8 [[KERNELONE_ACCTMP3:%.*]], [[TYPE0]] addrspace(1)* align 16 [[KERNELONE_ACCIN1:%.*]], %1* byval([[TYPE1]]) align 8 [[KERNELONE_ACCIN16:%.*]], [[TYPE0]] addrspace(1)* align 16 [[KERNELONE_ACCIN2:%.*]], [[TYPE0]] addrspace(1)* align 16 [[KERNELTWO_ACCOUT:%.*]], [[TYPE0]] addrspace(1)* align 16 [[KERNELTWO_ACCIN3:%.*]])
+; CHECK-SAME: (ptr addrspace(3) align 16 [[KERNELONE_ACCTMP:%.*]], ptr byval([[TYPE1:%.*]]) align 8 [[KERNELONE_ACCTMP3:%.*]], ptr addrspace(1) align 16 [[KERNELONE_ACCIN1:%.*]], ptr byval([[TYPE1]]) align 8 [[KERNELONE_ACCIN16:%.*]], ptr addrspace(1) align 16 [[KERNELONE_ACCIN2:%.*]], ptr addrspace(1) align 16 [[KERNELTWO_ACCOUT:%.*]], ptr addrspace(1) align 16 [[KERNELTWO_ACCIN3:%.*]])
 ; CHECK:  entry:
-; CHECK:    [[KERNELONE_ACCTMP31_SROA_0_0__SROA_IDX:%.*]] = getelementptr inbounds [[TYPE1]], %1* [[KERNELONE_ACCTMP3]], i64 0, i32 0, i32 0, i64 0
-; CHECK:    [[KERNELONE_ACCTMP31_SROA_0_0_COPYLOAD:%.*]] = load i64, i64* [[KERNELONE_ACCTMP31_SROA_0_0__SROA_IDX]], align 1
+; CHECK:    [[KERNELONE_ACCTMP31_SROA_0_0__SROA_IDX:%.*]] = getelementptr inbounds [[TYPE1]], ptr [[KERNELONE_ACCTMP3]], i64 0, i32 0, i32 0, i64 0
+; CHECK:    [[KERNELONE_ACCTMP31_SROA_0_0_COPYLOAD:%.*]] = load i64, ptr [[KERNELONE_ACCTMP31_SROA_0_0__SROA_IDX]], align 1
 ; CHECK:    [[TMP0:%.*]] = urem i64 [[KERNELONE_ACCTMP31_SROA_0_0_COPYLOAD]], 16
-; CHECK:    [[ADD_PTR_I_I:%.*]] = getelementptr inbounds [[TYPE0]], [[TYPE0]] addrspace(3)* [[KERNELONE_ACCTMP]], i64 [[TMP0]]
+; CHECK:    [[ADD_PTR_I_I:%.*]] = getelementptr inbounds [[TMP02:%.*]], ptr addrspace(3) [[KERNELONE_ACCTMP]], i64 [[TMP0]]
```

Comment:
Were these new checks created manually or by an LLVM script? If they were created manually, can we change `TMP02` to `TYPE...` to indicate that this is a type?

Reply:
Done.